KomodoPlatform/komodo-wallet-desktop
# Enhance/time sync warning

closes https://github.com/KomodoPlatform/komodo-wallet-desktop/issues/1857

To test:
A
- login, then modify your system time to be out of sync by a couple of minutes.
- Dex tab in sidebar should become disabled, and show a tooltip when hovering over
- If on dex page, it should kick you to portfolio page

B
- set your system time back to correct global time
- Dex tab in sidebar should once again be active
